xml/System.IO/DirectoryInfo.xml

```diff
@@ -3177,9 +3184,19 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the original path that was passed by the user.</summary>
+        <summary>Returns the original path that was passed by the user. Use the <see cref="P:System.IO.DirectoryInfo.FullName" /> or <see cref="P:System.IO.DirectoryInfo.Name" /> properties for the full path or file/directory name.</summary>
         <returns>The original path that was passed by the user.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+[!IMPORTANT]
+
+> The string returned by the <xref:System.IO.DirectoryInfo.ToString%2A> method represents path that was passed to the constructor. When you create a <xref:System.IO.DirectoryInfo> object using the constructors, the <xref:System.IO.DirectoryInfo.ToString%2A> method returns the fully qualified path. However, there are cases where the string returned by the <xref:System.IO.DirectoryInfo.ToString%2A> method does not represent the fully qualified path. To ensure consistent behavior across versions and explicit intent, call `ToString` on the <xref:System.IO.DirectoryInfo.Name> or <xref:System.IO.DirectoryInfo.FullName> of the current <xref:System.IO.DirectoryInfo> instance.
```

Comment:
```suggestion
> The string returned by the <xref:System.IO.DirectoryInfo.ToString%2A> method represents the path that was passed to the <xref:System.IO.DirectoryInfo> constructor. Instead of calling the `ToString` method, you should retrieve the value of either of the following properties, depending on your intent:
- <xref:System.IO.DirectoryInfo.Name>, to get the name of the directory, without any parent path information.
- <xref:System.IO.DirectoryInfo.FullName>, to get the fully qualified path of the directory.
```
This is contradictory; if it returns the string passed to the constructor (which the source indicates that it does), then it returns the fully qualified path only if one was provided to the constructor. I've suggested a revision.  

This can also be adapted for the remaining ToString methods.

Reply:
I see what you mean, thanks for the suggestion. I'll adapt it to the other ToString methods.